HIP/hip-1064.md

```diff
@@ -0,0 +1,165 @@
+---
+hip: 1064
+title: Daily Rewards For Active Nodes
+author: Neeharika Sompalli <@Neeharika-Sompalli>
+working-group: Leemon Baird (@lbaird), Richard Bair <@rbair23>, Michael Tinker <@tinker-michaelj>
+requested-by: Hashgraph
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+last-call-date-time: 2024-04-08T07:00:00Z
+created: 2024-10-17
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/1063
+updated: 2024-10-17
+---
+
+## Abstract
+
+This HIP proposes a reward mechanism that will incentivize nodes to remain active on the network. The definition of active node is described below. The minimum reward per day for nodes will be a configurable property decided by the council. This minimum reward may be adjusted based on fees already paid throughout the day.
+
```

Comment:
In the context of Hiero, we should reword "by the council" to mean "by the governance of the deployed network", or something similar.

Reply:
Changed. Thanks